website/blog/2025-11-13-ingress-nginx-update/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Update on ingress-nginx"
+date: "2025-11-13"
+description: "Azure Kubernetes Service's response to ingress-nginx retirement and recommended migration paths for AKS customers" 
+authors: ["ahmed-sabbour"]
+tags:
+  - app-routing
+  - istio
+  - traffic-management
+---
+
+The [Kubernetes SIG Networking](https://github.com/kubernetes/community/blob/master/sig-network/README.md) and ingress-nginx maintainers have [announced](https://www.kubernetes.dev/blog/2025/11/12/ingress-nginx-retirement/) the upcoming retirement of the Ingress NGINX project, with maintenance ending in **March 2026**.
```

Comment:
Should we call out why this was announced just a debrief say concisely from here https://www.kubernetes.dev/blog/2025/11/12/ingress-nginx-retirement/ - History and Challenges
The breadth and flexibility of Ingress NGINX has caused maintenance challenges. Changing expectations about cloud native software have also added complications. What were once considered helpful options have sometimes come to be considered serious security flaws, such as the ability to add arbitrary NGINX configuration directives via the “snippets” annotations. Yesterday’s flexibility has become today’s insurmountable technical debt.

Reply:
Also upstream page says Best-effort maintenance will continue until March 2026 - this should be called out no as best effort is not equal to maintenance.?

Reply:
on the first comment, the link gives context on the motivations behind the announcement. on second comment, we can probably add 'best-effort' to be consistent with upstream annoucement, rest looks good to me